src/Table/Table.component.tsx

```diff
@@ -268,6 +266,9 @@ export class Table extends React.Component<Props> {
       theme,
       ...props
     } = this.props;
+
+    console.log('rowsText = ', this.props.rowsText);
```

Comment:
Normal to have console.log in here?

Reply:
@cachapman I used to trust eslint checks but it won't work for cast-ui

Reply:
No worries, hence why we have peer reviews :-)